dubbo-config/dubbo-config-spring/src/main/resources/META-INF/compat/dubbo.xsd

```diff
@@ -1153,31 +1153,6 @@
         </xsd:complexContent>
     </xsd:complexType>
 
-    <xsd:complexType name="annotationType">
```

Comment:
why remove it from compat xsd? I think we cannot remove AnnotationBean, instead, it should be moved into compat package

Reply:
I think `AnnotationBean` has been completely replaced by @EnableDubbo and @DubboComponentScan, so everything related to `AnnotationBean` can be removed now.

We need @mercyblitz to confirm if the new mechanism can work without AnnotationBean.

Reply:
This schema file could be removed safely